pkg/sloop/server/server.go

```diff
@@ -111,7 +114,7 @@ func RealMain() error {
 		LeftBarLinks:     conf.LeftBarLinks,
 		CurrentContext:   displayContext,
 	}
-	err = webserver.Run(webConfig, tables)
+	err = webserver.Run(webConfig, tables, db)
```

Comment:
tables has a db parameter in it you can use

Reply:
fixed